pytest.ini

```diff
@@ -1,11 +1,14 @@
 [pytest]
 addopts = --verbose -ra
```

Comment:
If `--runslow` is needed to successfully run tests (per `build.yml`) then please add it here instead.

Reply:
Addressed here https://github.com/cisagov/con-pca-api/pull/632/commits/1e50c0a3374b14bdcbf370272be82f29a0d27d36